janitor/utils.py

```diff
@@ -1040,16 +1036,16 @@ def _data_checks_pivot_longer(
 ):
 
     """
-    This function raises errors if the arguments have the wrong python type,
-    or if an unneeded argument is provided. It also raises errors for some
-    other scenarios(e.g if there are no matches returned for the regular
-    expression in `names_pattern`, or if the dataframe has MultiIndex
-    columns and `names_sep` or `names_pattern` is provided).
+    This function raises errors if the arguments have the wrong python
+    type, or if an unneeded argument is provided. It also raises errors
+    for some other scenarios (e.g if there are no matches returned for
+    the regular expression in `names_pattern`, or if the dataframe has
+    MultiIndex columns and `names_sep` or `names_pattern` is provided).
```

Comment:
@loganthomas since we're changing the doctoring, can we use SEMBR on this doctoring instead please? https://sembr.org